python/delta/tests/test_deltatable.py

```diff
@@ -1187,6 +1188,41 @@ def test_protocolUpgrade(self) -> None:
         with self.assertRaisesRegex(ValueError, "writerVersion"):
             dt.upgradeTableProtocol(1, {})  # type: ignore[arg-type]
 
+    def test_addFeatureSupport(self) -> None:
+        try:
+            self.spark.conf.set('spark.databricks.delta.minWriterVersion', '2')
```

Comment:
Nit: Since we are checking `minReaderVersion` and `minWriterVersion`, we should also set `minReaderVersion` first then `minWriterVersion` instead of other way around.